webapps-schema/src/main/resources/schema/elasticsearch/create/index/operate-process.json

```diff
@@ -48,6 +48,12 @@
       "formId": {
         "type": "keyword"
       },
+      "formKey": {
```

Comment:
:thinking: Do we really need the `isFormEmbedded` field? The `formKey` seems enough to discriminate the type of form a process has.